getnacelle/nacelle-js
# feat(shopify-checkout): support non GIDs for variantIds

## WHY are these changes introduced?
Story [LS-1478](https://nacelle.atlassian.net/browse/LS-1478)

## WHAT is this pull request doing?

1. Adds a utility method for turning variantIds into valid shopify GIDs. This lets shopify checkout support v2 which currently strips off the `gid://` text before the id.
2. Uses that utility method in `cartItemsToCheckoutItems.ts` to convert each line item's variant Id to a `gid` if it wasn't already one.


## How to Test

1. Pull this PR locally
2. Run `npm run test:ci`

**Note** it'd probably be good to test this in an example project too to make sure that it works. Will update with instructions once I've gotten them.
